deploy/load_mock_data.js

```diff
@@ -0,0 +1,148 @@
+
+const MongoClient = require('mongodb').MongoClient;
+const assert = require('assert');
+
+const user = 'your_user';
+const pass = encodeURIComponent('your_password');
+const dbName = 'yourdbname';
+
+// Connection URL
+const url = `mongodb://${user}:${pass}@${dbName}.documents.azure.com:10255/?ssl=true`;
+
+// Database Name
+
+// Use connect method to connect to the server
+MongoClient.connect(url, function (err, client) {
+  assert.equal(null, err);
+  console.log("Connected successfully to server");
+
+  const db = client.db(dbName);
+  db.collection("ordering").insertMany(
+    [
+      {"skuNumber" : "LIG-0001", "description" : "Helogen Headlights (2 Pack)", "price" : 38.99, "inventory" : 10, "leadTime" : 3},
```

Comment:
leadTime got removed from the web app, so this might not be the correct data to populate

Reply:
do we have any other mock data to use instead?